tests/default.test.mjs

```diff
@@ -12,6 +12,8 @@ const appName = 'fancy-app-in-test';
 describe('basic functionality', function () {
   let tmpDir;
 
+  let emberCli = join(__dirname, '../node_modules/ember-cli/bin/ember');
```

Comment:
you can't depend on this location. You would need to do something like require.resolve to get the full path and then use that.

Reply:
Can do!